### ISSUE TITLE
Couldn't find any help with that topic on every query

### ISSUE THREAD

**Issue by SaurusXI** [gleitz/howdoi#332]
Persistent issue on every query which means that functionality is completely impaired at the moment. 
Did a little digging by making the following modification to source - 
```python
def get_proxies():
    proxies = getproxies()
    print(f'Proxies {proxies}')
    ...
```
Which gives the following output - 
```
Proxies {}
Sorry, couldn't find any help with that topic
```

Looks like no proxies are being fetched by the `getproxies()` call which is creating the issue.

**Comment by gleitz** [comment]
I believe this was an issue with Google search results which should now be fixed in 2.0.12. Would you please update howdoi using pip and let me know if you now get results?

**Comment by dsh1705** [comment]
updated to 2.0.12 and still have the issue noted above.
